log4j-core/src/main/java/org/apache/logging/log4j/core/appender/rolling/action/IfLastModified.java

```diff
@@ -97,17 +91,24 @@ public String toString() {
         return "IfLastModified(age=" + age + nested + ")";
     }
 
+    /**
+     * @since 2.24.0
+     */
     @PluginFactory
     public static Builder newBuilder() {
         return new Builder();
     }
 
-    public static class Builder implements Supplier<IfLastModified> {
+    /**
+     * @since 2.24.0
+     */
+    public static final class Builder implements Supplier<IfLastModified> {
         private Duration age;
         private PathCondition[] nestedConditions;
         private Clock clock;
 
-        public Builder setAge(@PluginAttribute final Duration age) {
+        public Builder setAge(
+                @PluginAttribute @Required(message = "No age provided for IfLastModified") final Duration age) {
```

Comment:
Is this `message` really needed? I think the DI should be smart enough to generate that, right? Doesn't it?

Reply:
The default message is somehow generic: `The parameter is null or empty: age`.

The message I added is the same as in `2.x`.